e2e/tests/functional/plugins/plot/stackedPlot.e2e.spec.js

```diff
@@ -228,6 +232,45 @@ test.describe('Stacked Plot', () => {
     ).toContainText(swgC.name);
   });
 
+  test('Changing properties of an immutable child plot are applied correctly', async ({ page }) => {
+    await page.goto(stackedPlot.url);
+
+    // Go into edit mode
+    await page.getByLabel('Edit Object').click();
+
+    await page.getByRole('tab', { name: 'Config' }).click();
+
+    // Click on canvas for the 1st plot
+    await page.getByLabel(`Stacked Plot Item ${swgA.name}`).click();
+
+    // Expand config for the series
+    await page.getByLabel('Expand Sine Wave Generator').click();
+
+    // turn off alarm markers. Technically there should be a getBy locator to ensure we're selecting the correct alarm marker
+    await page.getByLabel('Alarm Markers').uncheck();
```

Comment:
Technically this should be a chained `getBy` locator to ensure we're selecting the correct alarm marker